packages/breadcrumbs/src/styled/StyledBreadcrumb.ts

```diff
@@ -6,19 +6,21 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import BreadcrumbStyles from '@zendeskgarden/css-breadcrumbs';
-import { retrieveComponentStyles, isRtl } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'breadcrumbs.list';
 
-export const StyledBreadcrumb = styled.ol.attrs(props => ({
-  'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(BreadcrumbStyles['c-breadcrumb'], {
-    // RTL
-    [BreadcrumbStyles['is-rtl']]: isRtl(props)
-  })
-}))`
-  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
+export const StyledBreadcrumb = styled.ol.attrs(() => {
+  return {
+    'data-garden-id': COMPONENT_ID,
+    'data-garden-version': PACKAGE_VERSION
+  };
+})`
+  ${props => `
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    font-size: ${props.theme.fontSizes.md};
+    direction: ${props.theme.rtl ? 'rtl' : 'ltr'};
```

Comment:
I recall CSS properties should be in a specific order. I used [this](https://github.com/brandon-rhodes/Concentric-CSS/blob/master/style.css) as a guide when ordering. Is there docs for ordering that I should refer to? Please let me know if the ordering here is wrong.

Reply:
Order is [concentric](https://rhodesmill.org/brandon/2011/concentric-css/) as defined by https://github.com/zendeskgarden/stylelint-config/blob/master/plugins/stylelint-order.js. Use `yarn lint:css` to check while you're getting used to the ordering.